common/src/transaction/ethereum/error.rs

```diff
@@ -51,6 +51,8 @@ pub enum EthError {
     Eip712Error(#[from] Eip712Error),
     #[error("Json parse error{0}")]
     JsonError(serde_json::Error),
+    #[error("Client Error: {0}")]
+    ClientError(reqwest::Error),
```

Comment:
should the binding udl be updated as well: https://github.com/crypto-com/defi-wallet-core-rs/blob/main/common/src/common.udl#L400 ?

Reply:
it seems it should: https://github.com/crypto-com/defi-wallet-core-rs/runs/7496321820?check_suite_focus=true#step:9:49 ?

Reply:
ok